### PR TITLE
Upgrading hugo version to 0.30.2

### LINKED ISSUE #16: Upgrade to HUGO v0.30.2
- On this update `make` [has been replaced](https://github.com/gohugoio/hugo/releases/tag/v0.30) by `mage` for building hugo sites.
- Alphine doesn't have `mage` asset so it's installed manually from [the official documentation](https://github.com/magefile/mage#installation).

### PR DESCRIPTION
**Pull request by davidpelayo**
Fixes #16.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,13 +1,16 @@
-FROM handcraftedbits/nginx-unit-webhook:2.6.5
+FROM handcraftedbits/nginx-unit-webhook:latest
 MAINTAINER HandcraftedBits <opensource@handcraftedbits.com>
 
-ARG HUGO_VERSION=v0.28
+ARG HUGO_VERSION=v0.30.2
 
 COPY data /
 
+ENV GOPATH /opt/gopath/ \
```
Comment (davidpelayo) [inline Dockerfile]: This is mistaken. My fault. The backslash shouldn't be there.

File: Dockerfile [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
 RUN apk update && \
-  apk add git go libc-dev make && \
-
+  apk add --no-cache git go libc-dev && \
```
Comment (davidpelayo) [inline Dockerfile]: Additionally, `--no-cache` should be removed.

File: Dockerfile [hunk 3]
```diff
@@ -1,13 +1,17 @@
-FROM handcraftedbits/nginx-unit-webhook:2.6.5
+FROM handcraftedbits/nginx-unit-webhook:latest
```
Comment (handcraftedbits) [inline Dockerfile]: Can you revert this (I like to use specific version numbers for my images)?  Then I call accept the PR.  Thanks!

Comment (davidpelayo) [reply Dockerfile]: Sure! I'll do it now!

**Comment by handcraftedbits** [comment]
Should be available in Docker Hub shortly.  Thanks for your contribution!

**Pull request merged**